### ISSUE TITLE
Firefox reports wrong timezone

### ISSUE THREAD

**Issue by johnbotris** [tutao/tutanota#2853]
When resistFingerprinting is set in about:config, then firefox doesn't report the user's actual timezone, causing any event made in the calendar to be made for the incorrect time. Currently we just suggest that the user uses the desktop client, but we might want to find a work around, like having an account timezone (this would come with it's own set of issues)

**Comment by mpfau** [comment]
we could sent current time from server (e.g. after websocket connect) and report to the user if the deviation is too big.


Mar 16, 2021, 11:46 by ***@***.***:

**Comment by charlag** [comment]
We should just let setting timezone manually. It's anyway asked by many people who travel a lot.

**Comment by bedhub** [comment]
I agree that we should set a timezone manually. I am not sure if it would be enough to have one timezone per application or if we should have timezone per calendar.

**Comment by brendonmackenzie** [comment]
Could we please get the ability within Tuta to manually set a timezone so we can correctly see calendar appointments and the like. Many of us like to use Firefox with resistFingerprinting enabled. 

Not having this feature plays havoc with schedules. 

![Image](https://github.com/user-attachments/assets/cf99c2b3-ba20-4a29-87d1-c3e23b0f32e9)
![Image](https://github.com/user-attachments/assets/c8383f04-32fe-43a0-a190-fcce90176d61)
